InvSee/Commands.cs

```diff
@@ -82,6 +92,7 @@ public static void DoInvSee(CommandArgs args)
 
 					PlayerData data;
 					string name = "";
+					int userid = 0;
```

Comment:
`userid` doesn't seem to be used anywhere... I don't see the point in defining it. Is this a remnant of unfinished work (perhaps that other "invsee-ception" issue you've mentioned)?